packages/odyssey-storybook/src/components/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -0,0 +1,151 @@
+  parameters: {
+    docs: {
+      page: RadioGroupMdx,
+    },
+  },
+  argTypes: {
+    variant: {
+      options: ["indeterminate", "determinate"],
+      control: { type: "radio" },
+    },
+    value: {
+      control: { type: "number" },
+    },
+  },
+  decorators: [MuiThemeDecorator],
+};
+
+const DefaultTemplate: Story = (args) => {
+  const {} = args;
+  return (
+    <FormControl>
+      <FormLabel id="demo-radio-buttons-group-label">Speed</FormLabel>
+      <RadioGroup
+        aria-labelledby="demo-radio-buttons-group-label"
+        defaultValue="Lightspeed"
+        name="radio-buttons-group"
+      >
+        <FormControlLabel
+          value="Lightspeed"
+          control={<Radio />}
+          label="Lightspeed"
+        />
+        <FormControlLabel
+          value="Warp speed"
+          control={<Radio />}
+          label="Warp speed"
+        />
+        <FormControlLabel
+          value="Ludicrous speed"
+          control={<Radio />}
+          label="Ludicrous speed"
+        />
+      </RadioGroup>
+    </FormControl>
+  );
+};
+
+export const Default = DefaultTemplate.bind({});
+Default.args = {};
+
+const HintTemplate: Story = (args) => {
+  const {} = args;
+  return (
+    <FormControl>
+      <FormLabel id="demo-radio-buttons-group-label">Speed</FormLabel>
+      <FormHelperText>
+        Select the speed at which you wish to travel.
+      </FormHelperText>
+      <RadioGroup
+        aria-labelledby="demo-radio-buttons-group-label"
+        defaultValue="Lightspeed"
+        name="radio-buttons-group"
+      >
+        <FormControlLabel
+          value="Lightspeed"
+          control={<Radio />}
+          label="Lightspeed"
+        />
+        <FormControlLabel
+          value="Warp speed"
+          control={<Radio />}
+          label="Warp Speed"
+        />
+        <FormControlLabel
+          value="Ludicrous speed"
+          control={<Radio />}
+          label="Ludicrous Speed"
+        />
+      </RadioGroup>
+    </FormControl>
+  );
+};
+
+export const Hint = HintTemplate.bind({});
+Hint.args = {};
+
+const ErrorTemplate: Story = (args) => {
+  const {} = args;
+  return (
+    <FormControl error>
+      <FormLabel id="demo-radio-buttons-group-label">Speed</FormLabel>
+      <RadioGroup
+        aria-labelledby="demo-radio-buttons-group-label"
+        defaultValue="Lightspeed"
+        name="radio-buttons-group"
+      >
+        <FormControlLabel
+          value="Lightspeed"
+          control={<Radio color="error" />}
+          label="Lightspeed"
+        />
+        <FormControlLabel
+          value="Warp speed"
+          control={<Radio color="error" />}
+          label="Warp speed"
+        />
+        <FormControlLabel
+          value="Ludicrous speed"
+          control={<Radio color="error" />}
+          label="Ludicrous speed"
+        />
+      </RadioGroup>
+      <FormHelperText error>
+        <span style={visuallyHidden}>Error:</span>This field is required.
```

Comment:
I tried using `Box` and `sx` as described [here](https://mui.com/system/screen-readers/), but ran into a TS error I couldn't debug. For now, I'm using their fallback suggestion.